src/routes/data.ts

```diff
@@ -45,7 +45,10 @@ router.post(
   wrapAsync(clinicalApi.getProgramClinicalSearchResults),
 );
 router.post('/program/:programId/clinical-errors', wrapAsync(clinicalApi.getProgramClinicalErrors));
-router.post('/program/:programId/clinical-tsv', wrapAsync(clinicalApi.getDonorDataByIdAsTsvsInZip));
+router.post(
+  '/program/:programId/clinical-tsv',
+  wrapAsync(clinicalApi.getSpecificClinicalDataAsTsvsInZip),
+);
```

Comment:
This route wasn't supposed to be updated to `getDonorDataByIdAsTsvsInZip` in the PR that added the `/donors/tsv` route. This fix restores the original handler.